ghcide/src/Development/IDE/GHC/Compat/Outputable.hs

```diff
@@ -49,7 +49,7 @@ import           GHC.Driver.Session
 import           GHC.Driver.Types                as HscTypes
 import           GHC.Types.Name.Reader           (GlobalRdrEnv)
 import           GHC.Types.SrcLoc
-import           GHC.Utils.Error                 as Err hiding (mkWarnMsg)
+import           GHC.Utils.Error                 hiding (mkWarnMsg)
```

Comment:
It was an error in `Compat` module (notice it also was not `qualified`) & that prevented code from compiling on according GHC.

Notice the use of imports/reexports in other CPP for other GHC versions:
https://github.com/haskell/haskell-language-server/blob/cc3c81f7803cbd0cd3169249e889e4cc058ce214/ghcide/src/Development/IDE/GHC/Compat/Outputable.hs

They do not have default module qualified.